src/EncompassRest/Contacts/BusinessContactLicense.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace EncompassRest.Contacts
+{
+    public sealed class BusinessContactLicense : IDirty
```

Comment:
Perhaps rename this to just `ContactLicense` to prevent confusion with having a `PersonalContactLicense` property of type `BusinessContactLicense` as well as a `BusinessContactLicense` of type `BusinessContactLicense`.

Reply:
i thought about that, but it is specific to businessContacts